### DISCUSSION TITLE
Ignoring dual and getting feasible solution

### DISCUSSION THREAD

**Discussion by bharswami** [coin-or/Ipopt#520] [Q&A] [answered]
I am using IPOPT for MATLAB ([OptiToolbox](https://inverseproblem.co.nz/OPTI/)). I am interested in only feasible solutions, so I make the objective function a constant and run the solver. But I find that IPOPT is still trying to get a dual feasible solution. Is there a way to make the solver ignore the dual problem completely? I tried setting a large dual_inf_tol but even that didn't help. Please suggest ideas.
Regards

**Comment by svigerske** [comment, marked as answer]
Try setting `tol` and `compl_inf_tol` to large values, too.
`tol` is the max of primal infeas, dual infeasibility, and complementarity violation in the scaled problem.

**Comment by bharswami** [comment]
Worked out. Thanks.

**Comment by bharswami** [comment]
Of course needed to set constr_viol_tol and acceptable_constr_viol_tol
also. Also, the substitute for ‘tol’ in OptiToolbox is acceptable_tol.
